__tests__/app.test.js

```diff
@@ -141,4 +138,36 @@ describe("Set of GET Tests", () => {
         });
     });
   });
+  describe("GET Comments By Article ID", () => {
+    test("should return all comments for a specific article_id", () => {
+      return supertest(app)
+        .get("/api/articles/5/comments")
+        .expect(200)
+        .then((response) => {
+          const comments = response.body;
+          expect(comments.length > 0).toBe(true);
+          expect(Array.isArray(comments)).toBe(true);
+        });
+    });
+    test("should return a Status Code: 404 for a non-existent article ID ", () => {
+      return supertest(app)
+        .get("/api/articles/888/comments")
+        .expect(404)
+        .then((response) => {
+          expect(response.body.msg).toBe(
+            "No Comments Found For This Article..."
+          );
+        });
+    });
+    test("should return a Status Code: 400 if passed article ID is not a number", () => {
```

Comment:
Nice error tests! 

Another test case to consider, what would I expect to be returned if an article has no comments?